bc_obps/registration/api/v2/_operators/operator_id.py

```diff
@@ -16,7 +17,7 @@
 
 @router.get(
```

Comment:
This is for the Administration > Operators > Operator ID page; it wasn't fetching all the info